docs/cmd/md/meroxa-secrets-create.md

```diff
@@ -0,0 +1,41 @@
+---
+createdAt: 
+updatedAt: 
+title: "meroxa secrets create"
+slug: meroxa-secrets-create
+url: /cli/cmd/meroxa-secrets-create/
+---
```

Comment:
I briefly took a look at a few other examples in this directory but don't see anything with this heading - is this copy-pasted from somewhere else? or did I stumble across some older examples perhaps?